### PR TITLE
Swagger spec for disk.json + compute.json 2018-06

### PR DESCRIPTION
**Pull request by carissasun**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json [hunk 1]
```diff
@@ -5180,6 +5183,7 @@
         "storageAccountType": {
           "$ref": "#/definitions/StorageAccountType",
           "description": "Specifies the storage account type for the managed disk. Possible values are: Standard_LRS, Premium_LRS, and StandardSSD_LRS."
+          "description": "Specifies the storage account type for the managed disk. UltraSSD_LRS can only be used for data disks."
```
Comment (lmazuel) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json]: Twice description

**Review by lmazuel — CHANGES_REQUESTED** [review]
Double key in JSON that preventing generation and further testing.

**Review by lmazuel — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json [hunk 2]
```diff
@@ -5313,6 +5314,15 @@
       },
       "description": "Specifies the storage settings for the virtual machine disks."
     },
+    "AdditionalCapabilities": {
+        "properties": {
+            "ultraSSDEnabled": {
```
Comment (lmazuel) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json]: This does not follow ARM guideline. Even if Gaurav signed-of, I checked with @veronicagg that you should get a suppression rule approved by ARM:
https://github.com/Azure/adx-documentation-pr/wiki/Swagger-Validation-Errors-Suppression

Comment (carissasun) [reply specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json]: @rohbat

**Review by lmazuel — APPROVED** [review]

**Pull request merged**